pkg/runtime/secretstores/local/wrapper.go

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright 2021 Layotto Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package secret
+
+import (
+	"os"
+	"strings"
+
+	"github.com/dapr/components-contrib/secretstores"
+)
+
+type wrapper struct {
+	prefix string
+	comp   secretstores.SecretStore
+}
+
+func (w *wrapper) Init(metadata secretstores.Metadata) error {
+	if metadata.Properties != nil {
+		metadata.Properties["secretsFile"] = w.prefix + metadata.Properties["secretsFile"]
+	}
+	return w.comp.Init(metadata)
+}
+
+func (w *wrapper) GetSecret(req secretstores.GetSecretRequest) (secretstores.GetSecretResponse, error) {
+	return w.comp.GetSecret(req)
+}
+
+func (w *wrapper) BulkGetSecret(req secretstores.BulkGetSecretRequest) (secretstores.BulkGetSecretResponse, error) {
+	return w.comp.BulkGetSecret(req)
+}
+
+func Wrap(component secretstores.SecretStore) secretstores.SecretStore {
+	return &wrapper{
+		comp:   component,
+		prefix: getPrefixConfigFilePath(),
+	}
+}
+
+func getPrefixConfigFilePath() string {
+	prefix := ""
+	for i, str := range os.Args {
+		if str == "-c" || str == "-config" {
```

Comment:
The `FIXME` said that we should get the startup parameters in main.go and you can take https://github.com/mosn/layotto/blob/9a314511de93b122461e048850e0e9bf75e265a7/cmd/layotto/main.go#L455 as an example.
Checking `-c` or `-config` here is not enough, since users can just `./layotto start` and layotto will use the default configuration path

Reply:
> The `FIXME` said that we should get the startup parameters in main.go and you can take
> 
> https://github.com/mosn/layotto/blob/9a314511de93b122461e048850e0e9bf75e265a7/cmd/layotto/main.go#L455
> 
> as an example.
> Checking `-c` or `-config` here is not enough, since users can just `./layotto start` and layotto will use the default configuration path

Is there means I can get configuration path by used `c.String("config")` in main.go to replace `if str == "-c" || str == "-config"`?

Reply:
Yes! you can get the path by `c.String("config")` in main.go and then put it in a common place in the memory

We need to review the code and find a suitable place to store the path

Reply:
how to store var in memory? Can you get me some ref?

Reply:
The easiest way is to store it in the main.go as a global variable. But this "global variable" way is not "elegant".
I suggest you read the startup code and try to pass the var to the runtime server without using global variable.
If you really can't find a way, then use global variable

Reply:
@MichaelDeSteven Hi ,are u still working on it?
Do you need help? I can help with this

Reply:
sorry that I cant find the method of store var in startup code. can you get me some tips?

Reply:
Sure. let me take a look